go.mod

```diff
@@ -1,45 +1,121 @@
 module github.com/giantswarm/aws-operator
 
-go 1.15
+go 1.17
 
 require (
 	github.com/aws/amazon-vpc-cni-k8s v1.10.2
 	github.com/aws/aws-sdk-go v1.43.1
 	github.com/dylanmei/iso8601 v0.1.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/giantswarm/apiextensions/v3 v3.39.0
+	github.com/giantswarm/apiextensions/v6 v6.0.0
 	github.com/giantswarm/backoff v1.0.0
 	github.com/giantswarm/badnodedetector v1.0.1
 	github.com/giantswarm/certs/v3 v3.1.1
 	github.com/giantswarm/ipam v0.3.0
-	github.com/giantswarm/k8sclient/v5 v5.12.0
-	github.com/giantswarm/k8scloudconfig/v11 v11.1.2
+	github.com/giantswarm/k8sclient/v5 v5.11.0
```

Comment:
would it be possible to use only one version of `k8sclient`?

Reply:
done 🙏🏻